SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBPhase2TimeReconstructor.cc

```diff
@@ -52,9 +52,10 @@ void EcalEBPhase2TimeReconstructor::process(std::vector<int> &addout,
     setInput(addout[i]);
 
     if (debug_) {
-      std::cout << "  EcalEBPhase2TimeReconstructor::process(std::vector<int> buffer_ " << std::endl;;
+      std::cout << "  EcalEBPhase2TimeReconstructor::process(std::vector<int> buffer_ " << std::endl;
```

Comment:
`std::cout` is not allowed in production code. Please use `MessageLogger` instead.

Reply:
@srimanob There is a boolean in front ! Set to false by default